### PR TITLE
Five Factorial

### PR DESCRIPTION
**Pull request by doughale**
returns 5*4*3*2*1 optimized at 500% normal factorial runtime

### CODE REVIEW

File: five.js [hunk 1]
```diff
@@ -99,6 +99,10 @@
     var pointFive = five() / (five() + five())
     return Math.pow(five(), pointFive) * pointFive + pointFive;
   };
+  five.factorial = function() {
+    // returns 5*4*3*2*1 optimized at 500% normal factorial runtime;
+    return 120;
```
Comment (devsnek) [inline five.js]: should return `five() * 24`

Comment (doughale) [inline five.js]: Didn't want to take the performance hit

Comment (jackdcrawford) [inline five.js]: #perfmatters

**Comment by jackdcrawford** [comment]
Nice! Thanks for the awesome contribution @doughale

**Pull request merged**